data/arweave/assets/65cXZ7nYYJlbSJWIGaqdIpL8llR2QOCJpKoS6qjvoHc/meta.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Another Money",
+  "symbol": "ANO",
+  "web": "https://another.money",
+  "supply": 320000000,
+  "email": "another.money@hns.is",
+  "product": true,
+  "publicTeam": true,
+  "trusted": true,
```

Comment:
remove product, publicTeam and trusted props

Reply:
Cybr labs is public team that is well known in dWeb domains sphere (handshake). So I think publicTeam deserves to be here. About product, okay, but we'll send PR which will add product to true after few days, maybe week.
Also what does trusted means?

Reply:
I don't see any linkedin, and you have very few twitter followers so I doubt it's as well-known as you say

Reply:
Even if it's not super popular dWeb team, it still stays public team. Not every team must have LinkedIn profile, And I'd not recommend you to value teams by twitter followers.

Reply:
I agree with other changes, but if I say there's no public team, it would just be lie

Reply:
I only see an angry 🐭  talking to me right now 😄  That doesn't qualify as a public team which makes it harder to rugpull

Reply:
Do you need the whole team to talk with you?

Reply:
I (and everyone else depending on this repo) need to see public profiles. private convos don't qualify

Reply:
Here's list of few Cybr Labs members if you were talking about it:
https://twitter.com/ibeholdtheman
https://twitter.com/shrunbr
https://twitter.com/_sh1Tt
https://twitter.com/Stanzo_89
https://twitter.com/pgs2007

Reply:
Or how should I prove that it's public team? 🤔

Reply:
I guess this is good enough, paste that list in a new comment and add your own twitter too